src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,49 @@
+.nav {
+
+  &__item {
+    list-style: none;
+    position: relative;
+    margin-right: 20px;
+  }
+
+  &__item:last-child {
+    margin-right: 0;
+  }
+
+  &__list {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    align-items: flex-end;
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+    height: 60px;
+    text-decoration: none;
+    text-transform: uppercase;
+    font-family: Roboto, Arial, Helvetica, sans-serif;
```

Comment:
You already have this in html
![image](https://user-images.githubusercontent.com/91411692/223073942-c4d7c21a-4f36-469b-b951-dec5928d1898.png)

```suggestion
```